packages/vendure-plugin-sort-by-popularity/src/popularity.controller.ts

```diff
@@ -1,15 +1,16 @@
 import { Controller, Get, Param } from '@nestjs/common';
+import { Mutation } from '@nestjs/graphql';
 import { Ctx, RequestContext } from '@vendure/core';
-import { Success } from '../../test/src/generated/admin-graphql';
 import { SortService } from './sort.service';
 @Controller('/order-by-popularity')
 export class OrderByPopularityController {
   constructor(private sortService: SortService) {}
   @Get('calculate-scores/:mychanneltoken')
+  @Mutation()
```

Comment:
This is only for GraphQL, and we are using a REST endpoint here, so I don't think we need this?